Source/Core/Core/MemoryWatcher.cpp

```diff
@@ -2,12 +2,22 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
+#ifdef __unix__
+
 #include <fstream>
 #include <iostream>
 #include <sstream>
 #include <unistd.h>
 
+#elif defined(_WIN32)
+
+#include <iostream>
```

Comment:
If it's needed for all platforms, why is it in the ifdef at all?

I'd recommend moving out the other C++ headers (fstream, sstream) too. IIRC, MSVC includes a huge pile of other headers from iostream, I wouldn't recommend relying on that.